monkey/monkey_island/cc/services/authentication_service/authentication_facade.py

```diff
@@ -26,6 +30,12 @@ def needs_registration(self) -> bool:
         """
         return not User.objects.first()
 
+    def revoke_all_user_tokens(self, user: User):
+        """
+        Revokes all tokens for a specific user
+        """
+        self._datastore.set_uniquifier(user)
```

Comment:
What can we unit test here? That it was called?

Reply:
That's about it. It's a low-value test. Your call on whether or not it's worth it.